Chummer/Program.cs

```diff
@@ -68,6 +68,11 @@ static void Main()
             //for some fun try out this command line parameter: chummer://plugin:SINners:Load:5ff55b9d-7d1c-4067-a2f5-774127346f4e
             PageViewTelemetry pvt = null;
             var startTime = DateTimeOffset.UtcNow;
+
+            //Both of these need to be set, before any windows are created. 
+            Application.SetCompatibleTextRenderingDefault(false);
+            Application.SetUnhandledExceptionMode(UnhandledExceptionMode.ThrowException);
+
```

Comment:
I was really unsure about this and tried to do it some other way, because I was unsure of what problems this could entail.

But I did it in the end, because if I don't call them earlier I always run into `InvalidOperationException`
"You can only call this method before the first window is created by your Windows Forms application.". As soon as I call `Create()` in the constructor of `CustomDataDirectoryInfo`.

I couldn't find why that exception occurs, because AFAICT I don't create any windows anywhere. My original workaround was to call create in every `get` accessor and check before that if the object was already created. But that felt incredibly wrong.
Some brief testing right now showed, that the error doesn't occur as long as they are before 
`SetProcessDPI(GlobalOptions.DpiScalingMethodSetting);`, so they can be in the global mutex.